omnigan/trainer.py

```diff
@@ -248,17 +248,19 @@ def setup(self):
 
         # Get different optimizers for each task (different learning rates)
         self.g_opt, self.g_scheduler = get_optimizer(
-            self.G, self.opts.gen.opt, tasks=self.opts.tasks
+            self.G, self.opts.gen.opt, self.opts.tasks, self.opts.train.epochs
         )
 
         if get_num_params(self.D) > 0:
-            self.d_opt, self.d_scheduler = get_optimizer(self.D, self.opts.dis.opt)
+            self.d_opt, self.d_scheduler = get_optimizer(
+                self.D, self.opts.dis.opt, self.opts.train.epochs
```

Comment:
Wouldn't it put tasks=self.opts.train.epochs in the function instead of iterations=self.opts.train.epochs like you want?

Reply:
you're right it's a typo thanks!